storage/client_split_test.go

```diff
@@ -412,6 +412,63 @@ func TestStoreZoneUpdateAndRangeSplit(t *testing.T) {
 	}
 }
 
+// TestMaxBytesChangeAndRangeSplit tests a scenario where a new zone config
```

Comment:
Correct me if i'm wrong, but I think the range split is triggered by the creation of the new zone config, and has nothing to do with the maxbytes setting.